lib/ffmpeg/d3d12/encoder.py

```diff
@@ -32,14 +32,15 @@ def profile(self):
   @property
   def qp(self):
     def inner(qp):
-      if self.codec in [Codec.VP8, Codec.VP9]:
-        return f" -global_quality {qp}"
-      if self.codec in [Codec.AV1]:
+      if self.codec in [Codec.AVC, Codec.VP8, Codec.VP9]:
+        return f" -q {qp}"
+      if self.codec in [Codec.HEVC, Codec.AV1]:
         if "ICQ" == self.rcmode:
           return f" -global_quality {qp}"
+        return f" -q {qp}"
       if self.codec in [Codec.MPEG2]:
         mqp = mapRangeInt(qp, [0, 100], [1, 31])
-        return f" -global_quality {mqp}"
+        return f" -q {mqp}"
       return " -qp {qp}"
```

Comment:
Is `-qp` a valid option in d3d12 plugins?  How about we `return f" -q {qp}"` by default, instead?

Reply:
@tong1wu please help a double check

Reply:
D3D12 is identical to VAAPI. So just use the same option as vaapi(-q).